spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/orm/jpa/HibernatePropertiesTests.java

```diff
@@ -135,9 +141,37 @@ void defaultDdlAutoIsNotInvokedIfHibernateSpecificPropertyIsSet() {
 				.run(assertDefaultDdlAutoNotInvoked("create"));
 	}
 
+	@ParameterizedTest
+	@MethodSource("validJpaDatabaseActionsToExpectedHbmValues")
+	void defaultDdlAutoIsNotInvokedIfJpaDatabaseActionPropertyIsSetToValidJpaValue(String jpaDatabaseAction,
```

Comment:
Of course, these tests all reflect the "Strict Approach" outlined [above](https://github.com/spring-projects/spring-boot/pull/25129/files#r575549829).